modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -220,25 +220,24 @@ class ApiClient {
           if (value is List && (match = _regList.firstMatch(targetType)) != null) {
             targetType = match[1]; // ignore: parameter_assignments
             return value
-              .map((v) => _deserialize(v, targetType, growable: growable))
+              .map<dynamic>((dynamic value) => _deserialize(value, targetType, growable: growable))
```

Comment:
this value name here is shadowing upper level variable with name 'value'. can you rename it to something non-conflicting? I think `v` was alright, or `mappedValue`?